content/en/tools/digital-currencies/defining/activating-token-definitions.md

```diff
@@ -0,0 +1,97 @@
+---
+date: '2023-04-24'
+lastmod: '2023-06-14'
+section_menu: tools
+menu:
+  tools:
+    name: "Activating Token Definitions"
+    weight: 400
+    parent: digital-currencies-token-definitions
+    identifier: digital-currencies-activating-token-definitions
+    description: "Digital Currencies documentation describing how to activate decommissioned token definitions via the GUI"
+title: "Activating Token Definitions"
+---
+
```

Comment:
```suggestion
## Activating Token Definitions
```
Not sure which header number it should be though.

Reply:
Don't think we need an initial heading as that automatically comes from the Title of the page. Let me know if this is otherwise.